docs/en/05_Releases_and_changelogs/cwp_2.3.0.md

```diff
@@ -34,10 +34,13 @@ can be made:
 "prefer-stable": true
 ```
 
-### Bootstrap 4
+### Major Theme Updates
 
-Version 3.0.0 of the Starter and Wātea themes, available with CWP 2.3.0, are updated to use Bootstrap 4.3. Please
-[see the Bootstrap migration guide](https://getbootstrap.com/docs/4.3/migration/) for Bootstrap specific changes. 
+Version 3.0.0 of the Starter and Wātea themes, available with CWP 2.3.0, are updated to use Bootstrap 4.x. Please
+[see the Bootstrap migration guide](https://getbootstrap.com/docs/4.3/migration/) for Bootstrap-specific changes.
+
+These updates also include an upgrade to Laravel Mix 4, along with other dependency upgrades (including Webpack 4 and
+Babel 7). If you have modified any build configuration, this may need adjustment for compatibility.
```

Comment:
I think the only way this would be a problem for people is if they're using a fork/copy of Watea that they've customised for their project, and it relied on a previous version of starter theme. Are there other cases you can think of?

Reply:
True, not that I can think of. I'm not sure how widespread forks of Starter / Wātea are, nor how likely the creators of those forks are to rebase their work on top of this new version, but I don't think there's any harm in mentioning that these key packages are receiving major bumps. I can drop the warning about the build config if that doesn't seem worthwhile.

Reply:
I think it's worth mentioning but maybe more in the case of projects that have cascading themes that rely on starter or watea

Reply:
I've tweaked the wording - let me know what you think.